Source/CronosPlayUnreal/Private/DefiWalletCoreActor.cpp

```diff
@@ -396,6 +405,9 @@ void ADefiWalletCoreActor::InitializeNewDevelopmentOnlyWallet(
     rust::cxxbridge1::String result =
         _coreWallet->get_address(CoinType::CryptoOrgMainnet, 0);
     output = UTF8_TO_TCHAR(result.c_str());
+    // zeroize mnemonics
+    std::memset((void *)result.data(), 0, result.size());
```

Comment:
`memset` may be removed by the compiler: https://en.cppreference.com/w/c/string/byte/memset

`memset_s` should be more reliable or `SecureZeroMemory`: https://docs.microsoft.com/en-us/previous-versions/windows/desktop/legacy/aa366877(v=vs.85)?redirectedfrom=MSDN

Reply:
> memset may be optimized away (under the [as-if](https://en.cppreference.com/w/c/language/as_if) rules) if the object modified by this function is not accessed again for the rest of its lifetime (e.g., [gcc bug 8537](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=8537)). For that reason, this function cannot be used to scrub memory (e.g., to fill an array that stored a password with zeroes). This optimization is prohibited for memset_s: it is guaranteed to perform the memory write.

Reply:
memset_s seems better, i'll check